src/common/utils/csv-download-utils.ts

```diff
@@ -0,0 +1,80 @@
+import { GetTransactionListTypeEnum } from '@stacks/blockchain-api-client';
+import {
+  AddressBalanceResponse,
+  AddressTransactionsWithTransfersListResponse,
+  AddressTransactionWithTransfers,
+} from '@stacks/stacks-blockchain-api-types';
+import dayjs from 'dayjs';
+import { InfiniteData, QueryClient } from 'react-query';
+import { microStxToStx, microToStacks, toRelativeTime } from '../utils';
+
+export type CSVDownloadObjectType = {
+  Date: string;
+  'Sent Amount': string | number;
+  'Sent Currency': string;
+  'Received Amount': string | number;
+  'Received Currency': string;
+  'Fee Amount': string | number;
+  'Fee Currency': string;
+  'Net Worth Amount': string | number;
+  'Net Worth Currency': string;
+  TxHash: string;
+};
+
+export const getTxnData = (
```

Comment:
maybe rename it to `getTxsCSVData`

Reply:
I'll covert this into a hook and change its name